backends/advanced/src/advanced_omi_backend/routers/modules/annotation_routes.py

```
@@ -0,0 +1,318 @@
+                # Update memory
+                try:
+                    memory_service = get_memory_service()
+                    await memory_service.update_memory(
+                        memory_id=annotation.memory_id,
+                        content=annotation.corrected_text,
+                        user_id=current_user.user_id,
+                    )
+                    logger.info(
+                        f"Applied suggestion to memory {annotation.memory_id}"
+                    )
+                except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)